<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CBT-2F-301" ignored="0" name="Test(failed)" tests="1" failures="1" timestamp="2021-04-08T08:37:52 IST" time="20.676" errors="0">
  <testcase name="validLoginLogout" time="10.993" classname="com_AcitTime_Test.ValidLoginLogoutTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com_AcitTime_Generic.BasePage.verifyTitle(BasePage.java:30)
at com_AcitTime_Test.ValidLoginLogoutTest.validLoginLogout(ValidLoginLogoutTest.java:30)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- validLoginLogout -->
</testsuite> <!-- Test(failed) -->
